test/models/concerns/smart_proxy_extensions_test.rb

```diff
@@ -69,6 +71,21 @@ def test_rhsm_url_pulp_primary
     def test_rhsm_url_pulp_mirror
       assert_includes @proxy_mirror.rhsm_url, ":8443/rhsm"
     end
+
+    def test_sync_container_gateway
```

Comment:
This test is no longer testing arguments like it used to.  I think we could still test the args by not stubbing out `:update_container_repo_list` and `:update_container_repo_list` and instead putting expectations on `ProxyAPI::ContainerGateway.new(url: self.url).repository_list` and `ProxyAPI::ContainerGateway.new(url: self.url).user_repository_mapping(user_repo_map)`.   Those methods could have their input args tested like the old test did.